hammerblade/torch/tests/test_select.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests on select
+04/22/2020 Lin Cheng (lc873@cornell.edu)
+"""
+
+import torch
+import random
+
+torch.manual_seed(42)
+random.seed(42)
+
+def _test_select(x, index):
+    h = x.hammerblade()
+    assert h[index].device == torch.device("hammerblade")
```

Comment:
If a select like `x[2,3]`, throwing in such a test might be useful.

Reply:
Good point! never thought about that ;)
Added!